README.md

```diff
@@ -15,6 +15,19 @@
 
 This library is a Python library to work with OpenEPD format.
 
+> ⚠️ **Version Warning**
+>
+> This application is currently developed in **two major versions** in parallel:
+>
+> - **v6.x (>=6.0.0)** — Stable and production-ready. Maintains support for Pydantic v1 and v2 through a compatibility layer.
+> - **v7.x (>=7.0.0)** — Public beta. Functional and aligned with Pydantic v2 natively, but experimental and may introduce breaking changes in **internal and integration interfaces**.
```

Comment:
Say something about "Only the latest 7.x contains all features and fixes from 6.x."
Say something about what we will do (yanking of prerelease versions) when 7.x become stable.